src/Login/index.tsx

```diff
@@ -30,11 +30,9 @@ export const LoginScreen = (props: LoginProps) => {
         .then((response) => {
           if (response.status >= 200 && response.status < 300) {
             return response.json();
-          } else {
-            return setErrorMessage(
-              'Something has gone wrong with the server, please retry later',
-            );
           }
+          if (response.status === 401)
+            return setErrorMessage("Your email and your password don't match");
```

Comment:
It is very disturbing the lack of `{}` on the `if` condition, especially because they are on the upper part. At first I thought it was a problem of indentation and a forgotten `}` ... 
I was going to do a brief on your linter. However, it must not be in `auto on save` mode, you still have `"` instead of `' ` :(

But in fact, the important problem is that you are missing a treatment of this answer : case where status does not match any of your two conditions. For example a status of 500. In this case and for the time being, the promise will never be resolved